src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -1321,6 +1327,12 @@ public void setMsqMaxSelectableChoices(int qnNumber, int value) {
         executeScript(String.format("$('#%s').val(%d);$('#%s').change();", id, value, id));
     }
 
+    public void fillMsqMaxSelectableChoices(int qnNumber, String value) {
+        WebElement inputBox = getMsqMaxSelectableChoicesBox(qnNumber);
+        String id = inputBox.getAttribute("id");
+        fillTextBox(browser.driver.findElement(By.id(id)), value);
```

Comment:
i think you can replace `browser.driver.findElement(By.id(id))` with `inputBox`?

Reply:
I don't know what I was thinking when I did that!